readthedocs/core/views/__init__.py

```diff
@@ -21,7 +21,11 @@ class NoProjectException(Exception):
     pass
 
 
-class HealthCheckView(View):
+class HealthCheckView(CDNCacheControlMixin, View):
+
+    # Never cache this view, we always want to get the live response from the server.
```

Comment:
```suggestion
    # Never cache this view, we always want to get the live response from the server.
    # In production we should configure the health check to hit the LB directly,
    # but it's useful to be careful here in case of a misconfiguration.
```